charts/newrelic-infrastructure-beta/ci/test-values.yaml

```diff
@@ -0,0 +1,147 @@
+global:
+  licenseKey: 1234567890abcdef1234567890abcdef12345678
+  cluster: test-cluster
+
+common:
+  agentConfig:
+    eventQueueDepth: 6666666
+  config:
+    interval: 15s
+    timeout: 30s
+
+customAttributes: '{"new":"relic","loren":"ipsum"}'
+
+ksm:
+  enabled: true
+  annotations: {}
+  tolerations:
+    - operator: "Exists"
+      effect: "NoSchedule"
+    - operator: "Exists"
+      effect: "NoExecute"
+  extraEnv:
+    - name: ENV_VAR1
+      value: "var1"
+    - name: ENV_VAR2
+      value: "var2"
+  extraEnvFrom: []
+  # Mount additional volumes into the pod.
+  extraVolumes: []
+  # Mount additional volume Mounts into the pod.
+  extraVolumeMounts: []
+  resources:
+    limits:
+      memory: 300M
+    requests:
+      cpu: 100m
+      memory: 150M
+
+# K8s DaemonSets update strategy.
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 1
+
+enableProcessMetrics: "false"
+logFile: /var/log/nr-infra.log
```

Comment:
This breaks in unprivileged mode, I'd rather get rid of it.

Reply:
I've just removed it.